### ISSUE TITLE
Wrong exception

### ISSUE THREAD

**Issue by ylecuyer** [jgorset/facebook-messenger#107]
When using the example in the readme : 
```
message.reply(
  attachment: {
    type: 'image',
    payload: {
      url: 'http://sky.net/visual-aids-for-stupid-organisms/pig.jpg'
    }
  }
)
```

The returned exception is wrong: 

```
Facebook::Messenger::Bot::**RecipientNotFound**: (#100) Failed to fetch the file from the url
```

**Comment by espenhogbakk** [comment]
@ylecuyer looks like that url is not valid. If you try to load it in the browser it doesn't load. Actually the whole domain sky.net is not available it seems.

Try with another url that you can verify works, like this: http://i.imgur.com/vG98twU.jpg

**Comment by ylecuyer** [comment]
I know that the URL is wrong. The issue here is with the exception type

**Comment by ylecuyer** [comment]
Same thing for

`Facebook::Messenger::Bot::RecipientNotFound at /
(#100) Invalid keys "playload" were found in param "message[quick_replies][0]".`

**Comment by jgorset** [comment]
Ha, right! I'll reopen this, then.

**Comment by sorich87** [comment]
The issue was fixed in #99. `RecipientNotFound` class doesn't exist anymore.

**Comment by jgorset** [comment]
Nice @sorich87!
